src/cosmos/gov/v1beta1/gov.ts

```diff
@@ -811,119 +951,133 @@ export const DepositParams = {
 
   toJSON(message: DepositParams): unknown {
     const obj: any = {};
+
     if (message.minDeposit) {
       obj.minDeposit = message.minDeposit.map((e) => (e ? Coin.toJSON(e) : undefined));
     } else {
       obj.minDeposit = [];
     }
-    message.maxDepositPeriod !== undefined &&
-      (obj.maxDepositPeriod = message.maxDepositPeriod
-        ? Duration.toJSON(message.maxDepositPeriod)
-        : undefined);
```

Comment:
here is a difference, but it only sets the result to `undefined` which is equivalent.